ciw/tests/test_node.py

```diff
@@ -637,6 +639,35 @@ def test_server_utilisation_with_schedules(self):
         recs = Q.get_all_records()
         self.assertEqual(Q.transitive_nodes[0].server_utilisation, 21.0/37.0)
 
+    def test_server_utilisation_with_wrapup(self):
+        N = ciw.create_network(
+            Arrival_distributions=[['Exponential',7.14]],
+            Service_distributions=[['Exponential',0.04]],
+            Number_of_servers=[70]
+        )
+        ciw.seed(1)
+        Q = ciw.Simulation(N)
+        Q.simulate_until_max_time(168)
+        for srvr in Q.transitive_nodes[0].servers:
+            self.assertGreaterEqual(srvr.total_time, srvr.busy_time)
```

Comment:
Also, this doesn't have the check of the actual bug does it?

Reply:
yes. The problem was `srvr.busy_time` was larger then `srvr.total_time` because I was also adding the final customers' waiting time to the busy time.

Reply:
Ah yes! I see that now :+1: